samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: cnfw.cisco.com/v1
```

Comment:
regarding this directory structure...

would it make more sense to have it split on service-chained vs non service chain? example

* `samples/service-chaining/`
* `samples/service-chaining/vpn`
* `samples/service-chaining/cntd`
* `samples/default/`
* `samples/default/vpn`  <-- all the existing samples could move under this one

Reply:
After discussion with Santosh Kodali yesterday, we thought current structure is better one.

Reply:
what is the reasoning? if i'm someone looking to set up end to end service chaining, I'm bouncing around a lot of directories, especially considering where the CNTD stuff was merged to a couple days ago

Reply:
the proposed project structure makes sense to me

Reply:
Done